H2TauTau/scripts/harvest.py

```diff
@@ -243,11 +225,8 @@ def harvest(src, subdir_pattern='*', tgz_pattern='*', skim=False, apply_ff=False
         print ds.dest
         ds.unpack()
         ds.hadd()
-        if skim:
-            ogpath = '{}/0000/{}/NtupleProducer/tree.root'.format(ds.dest,ds.dest[:ds.dest.find('/')])
-            newpath = '{}/tree.root'.format(ds.dest[:ds.dest.find('/')])
-            skimntuple(ogpath,newpath)
-            os.system('rm -rf {}'.format(ds.dest))
+        os.system('cp {dirname}/0000/{compname}/NtupleProducer/tree.root {dirname}/tree.root'.format(dirname=ds.dest,compname=ds.dest[ds.dest.find('/'):]))
```

Comment:
This could work if there is not too many jobs but sometimes you get
```
$ ls $dirname
0000
0001
...
```
so maybe one may re-think these lines.

Reply:
I have not ever experienced such a case, could you share an example? Would you know what are the thresholds/rules that lead to this case?

Reply:
It happens when the amount of crab jobs exceeds 1000.

Reply:
ok thanks for the information, I will update and push a fix

Reply:
ok I have pushed a fix, unfortunately I haven't tested it in real conditions as I don't think I have an example of this case. Maybe you could?
Also I don't really have time to work on this anymore, but I think you got the idea of the code design. This latest fix is not very elegant so don't hesitate to submit a fix if you have a better way of coding it.